_data/definitions.yml

```diff
@@ -0,0 +1,14 @@
+BOVS:
+  Node: A vertex in a BOVS diagram. Nodes function like nouns, representing Parties.
+  Arrow: An edge in a BOVS diagram. Arrows function like verbs, connecting Nodes to show relationships.
+  Chain: A series of Nodes continuously connected by Arrows is called a Chain. The "start" of the Chain is the first Node and the "end" is the last, based on the direction of the Arrows.
+  Party: General term for any kind of thing (e.g. Person, Company) that can be involved in a beneficial ownership situation.
+  Beneficiary: Any Party that has ownership or control of another Party. The Beneficiary at the start of a Chain is the "ultimate" Beneficiary.
```

Comment:
Flagging this definition for @ScatteredInk to check - as I know there have been some questions about use of the term 'ultimate' in beneficial ownership in the past. 

My sense is that this is a justified use of the idea of 'ultimate' beneficial ownership, but I want to double check on this. For context, @ScatteredInk this appears at https://deploy-preview-39--open-ownership.netlify.app/visualisation/core/definitions/

Reply:
Ultimate is now gone, so resolving this.